imagepicker/src/main/java/com/esafirm/imagepicker/features/ImageFileLoader.java

```diff
@@ -85,6 +87,9 @@ public void run() {
 
                     File file = makeSafeFile(path);
                     if (file != null && file.exists()) {
+                        if(exlucedImages != null && exlucedImages.contains(file))
```

Comment:
Hey i'm not sure about this. Is the `equal()` is triggering path comparison for `File`?

Reply:
https://docs.oracle.com/javase/7/docs/api/java/io/File.html#equals(java.lang.Object)
But can be changed to a loop that checks the absolute path.

Reply:
Ah OK i think it's suffice 👍

Reply:
Ok, made all the changes.